src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public int getRandomBall() {
+        Random random = new Random();
```

Comment:
```suggestion
public class Lottery {
    private ColorSupplier colorSupplier = new ColorSupplier();
    private Random random = new Random();
    
    public int getRandomBall() {
```
lets make those objects global

Reply:
global - need to make getters and setters?